spiketools/spatial/utils.py

```diff
@@ -10,18 +10,41 @@ def get_pos_ranges(positions):
 
     Parameters
     ----------
-    positions : 2d array
+    positions : 1d or 2d array
         Position data.
 
     Returns
     -------
     ranges : list of list of float
         Ranges for each dimension in the spatial data.
+
+    Examples
+    --------
+    Get 2D position ranges for:
+    (x, y) = (1.5, 6.5), (2.5, 7.5), (3.5, 8.5), (5, 9).
+
+    >>> position = np.array([[1.5, 2.5, 3.5, 5], [6.5, 7.5, 8.5, 9]])
+    >>> get_pos_ranges(position)
+    [[1.5, 5.0], [6.5, 9.0]]
+
+	Get 1D position ranges for:
+    x = 1.5, 2.5, 3.5, 5.
+
+    >>> position = np.array([1.5, 2.5, 3.5, 5])
+    >>> get_pos_ranges(position)
+    [[1.5, 5.0]]
     """
 
     ranges = []
-    for dim in range(positions.shape[0]):
-        ranges.append([np.min(positions[dim, :]), np.max(positions[dim, :])])
+
+    # 2+d case
+    if (len(positions.shape) > 1):
```

Comment:
numpy arrays have an `ndim` field, which is a more explicit attribute for this, so this could be `if position.ndim >=2:`